src/main/scala/de/upb/cs/swt/delphi/instanceregistry/connection/Server.scala

```diff
@@ -9,13 +9,13 @@ import akka.stream.ActorMaterializer
 import akka.stream.scaladsl.{Flow, Sink, Source}
 import de.upb.cs.swt.delphi.instanceregistry.io.swagger.client.model.InstanceEnums.ComponentType
 import de.upb.cs.swt.delphi.instanceregistry.io.swagger.client.model._
-import de.upb.cs.swt.delphi.instanceregistry.{AppLogging, Registry, RequestHandler}
+import de.upb.cs.swt.delphi.instanceregistry._
```

Comment:
Is this change to the import statement necessary ?

Reply:
I have fixed it. Please check.